src/grid.js

```diff
@@ -100,14 +120,62 @@ Grid.prototype = {
 
   // Render entire grid
   render: function() {
-    return this.renderHeader().renderBody();
+    return this.calculateWidth().renderHeader().renderBody();
+  },
+
+  // Calculate column width
+  assignWidth: function() {
+    var borderWidth = 1;
+    var rowWidth = this.$width - (2 * borderWidth);
+    if (this.$height) {
+      this.$table.addClass('waffle-fixedheader')
+                 .attr($$width(this.$width));
```

Comment:
You should not override size attribute, it could be used for other things... Instead, add inline style (check jQuery / angular jqLite api).